pkg/instance/tshark.go

```diff
@@ -40,13 +42,21 @@ func (i *Instance) createTsharkCollectorInstance(ctx context.Context) (*Instance
 	if err := tsc.Commit(); err != nil {
 		return nil, err
 	}
-	if err := tsc.SetCPU(tsharkCollectorCPU); err != nil {
+	if err := tsc.SetCPU(resource.MustParse(tsharkCollectorCPU)); err != nil {
```

Comment:
same comment about moving `tsharkCollectorCPU` to a var so we can validate at compile.